examples/evaluate_eval_assist_direct_user_criteria_no_catalog.py

```diff
@@ -0,0 +1,76 @@
+from unitxt import get_logger
+from unitxt.api import create_dataset, evaluate
+from unitxt.eval_assist_constants import (
+    CriteriaWithOptions,
+    EvaluatorNameEnum,
+)
+from unitxt.eval_assist_llm_as_judge_direct import EvalAssistLLMAsJudgeDirect
+from unitxt.inference import LiteLLMInferenceEngine
+from unitxt.text_utils import print_dict
+
+logger = get_logger()
+
+criteria = CriteriaWithOptions.from_jsons(
+    s="""
+{
+    "name": "Temperature in Fahrenheit and Celsius",
+    "description": "In the response, if there is a numerical temperature present, is it denominated in both Fahrenheit and Celsius?",
+    "options": [
+        {
+            "name": "Yes",
+            "description": "The temperature reading is provided in both Fahrenheit and Celsius."
+        },
+        {
+            "name": "No",
+            "description": "The temperature reading is provided either in Fahrenheit or Celsius, but not both."
+        },
+        {
+            "name": "Pass",
+            "description": "There is no numerical temperature reading in the response."
+        }
+    ],
+    "option_map": {"Yes": 1.0, "No": 0.5, "Pass": 0.0}
+}
+"""
```

Comment:
can we have this working by a `from_dict` function? it seems like from json is not suitable in this case and is unnatural python code

Reply:
There is a `from_obj()` class method that returns a `CriteriaWithOptions` object from a dict. I didn't use the `from_dict()` method because it needs the dict to have a `__type__` attribute. If not, it throws:
`unitxt.artifact.MissingArtifactTypeError: Missing '__type__' parameter`.